src/components/Slider/__tests__/Slider-test.js

```diff
@@ -1,76 +1,23 @@
-import React from 'react';
-import expect from 'expect';
-import TestUtils from 'react-addons-test-utils';
-import expectJSX from 'expect-jsx';
-import {RawSlider as Slider} from '../Slider';
-import Nouislider from 'react-nouislider';
-expect.extend(expectJSX);
-
-describe('Slider', () => {
-   // to ensure the global.window is set
 
-  let renderer;
-  let props;
-
-  beforeEach(() => {
-    const {createRenderer} = TestUtils;
-    renderer = createRenderer();
-
-    props = {
-      range: {min: 0, max: 5000},
-      format: {to: () => {}, from: () => {}},
-      start: [0, 0],
-    };
-  });
+import React from 'react';
+import renderer from 'react-test-renderer';
 
-  it('should render <NouiSlider {...props} />', () => {
-    const out = render();
-    expect(out).toEqualJSX(
-      <Nouislider
-        animate={false}
-        behaviour="snap"
-        connect
-        cssPrefix="ais-range-slider--"
-        format={{to: () => {}, from: () => {}}}
-        onChange={() => {}}
-        pips={{
-          density: 3,
-          mode: 'positions',
-          stepped: true,
-          values: [0, 50, 100],
-        }}
-        range={props.range}
-        start={ props.start }
-      />
-    );
-  });
+import Slider from '../Slider';
 
-  it('should render <NouisLider disabled="true" /> when ranges are equal', () => {
-    props.range.min = props.range.max = 8;
-    const out = render();
-    expect(out).toEqualJSX(
-      <Nouislider
-        animate={ false }
-        behaviour="snap"
-        connect
-        cssPrefix="ais-range-slider--"
-        format={ {to: () => {}, from: () => {}} }
-        onChange={ () => {} }
-        pips={ {
-          density: 3,
-          mode: 'positions',
-          stepped: true,
-          values: [0, 50, 100],
-        } }
-        range={ {min: props.range.min, max: props.range.min + 0.0001} }
-        start={ props.start }
-        disabled
+describe('Slider', () => {
+  it('should render correctly', () => {
+    const tree = renderer.create(
+      <Slider
```

Comment:
Shallow rendering will only render the top component, so in that case it would have been the `<AutohideContainerHOC />`.

I just render it like this to prevent further regression with snapshot testing, it's a dummy test case. Look for `range-slider-test.js` on the widget itself where I test the rendering and the logic 👍